config/.env.sample

```diff
@@ -34,6 +34,12 @@ OVERWRITE_EXISTING_TOPICMODEL = 0
 # Defaults to 0
 OVERWRITE_EXISTING_QUESTIONS = 0
 
+# Minimum threshold for the video duration in seconds for processing.
+# Shorter videos might not have enough content to generate meaningful topics and questions.
+# Value must be >= 300s, but it is recommended to have it over 600s.
+# Defaults to 300.
+MINIMUM_VIDEO_DURATION = 300
```

Comment:
Frankly, I don't see the reason why this need to a `env` variable. We don't want a bunch of env variable. you can make this value as constant.

Reply:
This has been implemented in the latest commit.

What you said makes sense, I do not expect this value to change once set.